web/html/src/components/combobox.tsx

```diff
@@ -20,6 +22,9 @@ type ComboboxProps = {
   selectedId?: (number | null | undefined) | (string | null | undefined);
   onFocus?: () => void;
   onSelect: (value: ComboboxItem) => void;
+
+  /** Id for testing purposes */
+  "data-testid"?: string;
```

Comment:
Since we might wanna use this in more components as a good standard for testing, can we make this a separate global type and extend the types with it where needed?

Reply:
This is a simple change to make once we actually use it in more places than one, I wouldn't recommend abstracting it out unless we have somewhere else to put it as well.